cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -119,6 +119,14 @@ the argument will be set for all programs that use ``myTable``. In order to set
 program ``myMR``, the dataset argument can be nested inside the program scope as
 ``mapreduce.myMR.dataset.myTable.cache.seconds=30``.
 
+Overriding Logical Start Time with Runtime Arguments
+====================================================
+
+The runtime argument ``logicalStartTime`` can be used to override the logical start time of a Workflow, MapReduce,
```

Comment:
We should have a uniform way of naming runtime arguments that are recognizable by the platform? E.g. in workflow local dataset, it has runtime argument for keeping local dataset as `dataset.<dataset_name>.keep.local`. If we follow the same convention, the logical start time would be `logical.start.time`. I don't know if we have other runtime arguments like that. @anew comments?